presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraSession.java

```diff
@@ -428,11 +428,13 @@ private static void checkColumnNames(List<ColumnMetadata> columns)
 
     public ResultSet execute(String cql)
     {
+        log.debug("Execute cql: " + cql);
```

Comment:
Use %s to avoid paying the String fmt'ing cost when debug log is disabled.

```suggestion
        log.debug("Execute cql: %s", cql);
```

same in other cases